templates/quickstart-jira-dc.template.yaml

```diff
@@ -1035,114 +1006,51 @@ Resources:
                     - !Sub ["ATL_TOMCAT_REDIRECTPORT=${TomcatRedirectPort}", TomcatRedirectPort: !Ref TomcatRedirectPort]
                     - !Sub ["ATL_TOMCAT_SCHEME=${TomcatScheme}", TomcatScheme: !If [SSLScheme, https, http]]
                     - !Sub ["ATL_TOMCAT_SECURE=${TomcatSecure}", TomcatSecure: !If [SSLScheme, true, false]]
-                    - "ATL_APP_DATA_MOUNT_ENABLED=false"
-                    - "ATL_DB_NAME=jira"
-                    - "ATL_DB_USER=jira"
-                    - "ATL_ENABLED_PRODUCTS=Jira"
-                    - "ATL_ENABLED_SHARED_HOMES="
-                    - "ATL_JDBC_DRIVER=org.postgresql.Driver"
-                    - "ATL_JDBC_USER=atljira"
-                    - "ATL_NGINX_ENABLED=false"
-                    - "ATL_POSTGRES_ENABLED=false"
-                    - "ATL_RELEASE_S3_BUCKET=atlassian-software"
-                    - "ATL_RELEASE_S3_PATH=releases"
-                    - "ATL_SSL_SELF_CERT_ENABLED=false"
-            /etc/cfn/cfn-hup.conf:
-              content:
-                !Join
-                  - "\n"
-                  -
-                    - "[main]"
-                    - !Sub ["region=${Region}", Region: !Ref "AWS::Region"]
-                    - !Sub ["stack=${StackId}", StackId: !Ref "AWS::StackId"]
-              mode: "000400"
-              owner: root
-              group: root
-            /etc/cfn/hooks.d/cfn-auto-reloader.conf:
```

Comment:
Why are we removing cfn update completely? Wouldn't be better to rerun the playbook listed as a parameter? The playbook should be idempotent and this way people can rerun the playbook with new values.

Reply:
It wasn't clear to me that this should be the case. Every conversation I've had around updating templates assumes existing nodes are brought down first and new ones brought up after the update. It also raises questions about how the roles should behave when updated by ansible, e.g. should a change to the startup script cause Jira to restart?

Reply:
Perhaps, we address this later?